bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.html

```diff
@@ -1,31 +1,36 @@
-<div fxFlexFill>
+<div fxFlexFill fxLayout="column">
+  <ktb-notification-bar></ktb-notification-bar>
   <div class="container" fxFlex fxLayout="column">
-    <div class="mb-3 settings-section" fxFlex fxLayout="column">
-      <h2>Git upstream repository</h2>
-      <p class="mt-0 mb-3">Instructions, how to set your Git provider can be found in the
-        <a [href]="'/manage/git_upstream/' | keptnUrl" target="_blank">Git-based upstream documentation</a>
-      </p>
-      <form [formGroup]="gitUpstreamForm" class="mb-3" fxLayout="row" fxLayoutAlign="none end">
-        <dt-form-field class="mr-3">
-          <dt-label>Git remote URL</dt-label>
-          <input formControlName="gitUrl" type="text" required dtInput placeholder="https://git-repo.com/repo.git"/>
-          <dt-error>Must not be empty</dt-error>
-        </dt-form-field>
-        <dt-form-field class="mr-3">
-          <dt-label>Git user</dt-label>
-          <input formControlName="gitUser" type="text" required dtInput placeholder="Username"/>
-          <dt-error>Must not be empty</dt-error>
-        </dt-form-field>
-        <dt-form-field class="mr-3">
-          <dt-label>Token</dt-label>
-          <input formControlName="gitToken" (focus)="gitTokenControl.value.length > 0 && gitTokenControl.value.includes('*********') ? gitTokenControl.setValue('') : null" type="text" dtInput placeholder="Token"/>
-          <dt-error>Must not be empty</dt-error>
-        </dt-form-field>
-        <button [disabled]="gitUpstreamForm.invalid || !gitUpstreamForm.dirty || isGitUpstreamInProgress" (click)="setGitUpstream()" dt-button>
-          <dt-loading-spinner *ngIf="isGitUpstreamInProgress" aria-label="Setting Git upstream URL"></dt-loading-spinner>
-          Set Git upstream
-        </button>
-      </form>
+    <div [formGroup]="projectNameForm" *ngIf="isCreateMode" class="mb-3 settings-section">
+      <h2>Project name (mandatory)</h2>
+      <dt-form-field>
+        <dt-label>Project name</dt-label>
+        <input type="text" formControlName="projectName" required dtInput placeholder="e.g. sockshop"/>
```

Comment:
We need some additional validation for the project name.
`Keptn relies on the following conventions: start with a lower case letter, then lower case letters, numbers, and hyphens are allowed.`

Reply:
Added a validation with regex and a hint below the form field. Unit tests for the pattern are also added.